scripts.js

```diff
@@ -50,6 +52,104 @@ menuContactUs.addEventListener('click', () => {
+    popuptec: ['CSS', 'Html', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/golf.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+    btnName: 'btn1',
+    btnimg1: './pictures/TimeAndIcons/github.png',
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+  {
+    id: 1,
+    popuptitle: 'Multi-Post Stories',
+    popupdescription: 'This is the second A daily for this is a test and is test number 2',
+    popuptec: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/beach.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+    btnName: 'btn3',
+    btnimg1: './pictures/TimeAndIcons/github.png',
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+  {
+    id: 2,
+    popuptitle: 'Multi-Post Stories',
+    popupdescription: 'This is the third A daily for this is a test and is test number 3',
+    popuptec: ['CSS', 'HTML', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/desert.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+    btnName: 'btn4',
+    btnimg1: './pictures/TimeAndIcons/github.png',
+    btnimg2: './pictures/TimeAndIcons/github.png',
+  },
+];
+
+if (window.matchMedia('(max-width: 768px)').matches) {
+  document.getElementById('Portfolio-menu').innerHTML = arr1.map((items) => `<div class="card-works">  
+                <h2 class="card-works-title" id="card-works-title">${items.popuptitle}</h2>
+                <p class="card-works-description" id="card-works-description">${items.popupdescription}</p>
+                <ul class="card-works-tech">
+                  <li class="card-works-tech-items">${items.popuptec[0]}</li>
+                  <li class="card-works-tech-items">${items.popuptec[1]}/li>
+                  <li class="card-works-tech-items">${items.popuptec[2]}</li>
+                  <li class="card-works-tech-items">${items.popuptec[3]}</li>
+                </ul>
+                <button class="card-works-button project-2" type="button">
+                  <a href="#popup">See Project</a>
+                </button>
+              </div>`).join('');
+} else {
+  document.getElementById('Portfolio-menu').innerHTML = arr1.map((items) => `<div class="card-works-alt background-2">
+                  <h2 class="card-works-title title-alt">${items.popuptitle}</h2>
+                  <p class="card-works-description description-alt">A${items.popupdescription}</p>
+                  <ul class="card-works-tech tech-alt">
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[0]}</li>
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[1]}</li>
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[2]}</li>
+                    <li class="card-works-tech-items tech-items-alt">${items.popuptec[3]}</li>
+                  </ul>
+                </div>`).join('');
+}
```

Comment:
- [ ] Great work here 👌 but I noticed only the first card is shown in `mobile-version`, Please make sure that all the cards appear and work as expected in both mobile and desktop versions 💯

Reply:
![image](https://user-images.githubusercontent.com/106117541/180569283-18f898ea-606f-49bd-97c3-23a2e0603507.png)